src/Raven.Server/Documents/ConflictsStorage.cs

```diff
@@ -110,30 +110,26 @@ internal GetConflictsPreviewResult GetConflictsPreviewResult(DocumentsOperationC
         {
             var table = context.Transaction.InnerTransaction.OpenTable(ConflictsSchema, ConflictsSlice);
 
-            var conflicts = new List<GetConflictsPreviewResult.ConflictPreview>();
-            foreach (var tvr in table.SeekBackwardFromLast(ConflictsSchema.FixedSizeIndexes[AllConflictedDocsEtagsSlice], skip))
+            var conflictsDictionary = new Dictionary<string, GetConflictsPreviewResult.ConflictPreview>();
+            foreach (var tvr in table.SeekBackwardFromLast(ConflictsSchema.FixedSizeIndexes[AllConflictedDocsEtagsSlice]))
```

Comment:
you will scan here through the entire table.
1. we need page size
2. we need skip

Skip is not the number of the unique Ids, but should be the number of scanned items.

Reply:
Done